tests/atomistics/master/test_murnaghan.py

```diff
@@ -90,7 +90,12 @@ def test_fitting_routines(self):
         murn._hdf5["output/equilibrium_volume"] = 448.4033384110422
         murn.status.finished = True
 
-        murn.plot(plt_show=False)
+        self.assertIsInstance(murn.plot(plt_show=False), matplotlib.axes.Axes)
```

Comment:
Note: Apparently calling `plt.show()` in our CI environment causes the Windows and MacOS tests to run forever (Linux tests seem to be fine).